docs/src/content/use_cases/specify-color.md

```diff
@@ -140,3 +141,535 @@ series: [
+      [{
+        legendHolderSelector: '#threshold_barLine_legend',
+        seriesRef: ['Max', 'Min', 'Average'],
+      }]);
+    // Manage window size changed
+    themeManager.manageChartResize(myChart, 'threshold_barLine_chart');
+    // Register the externalization of the tooltip/popup
+    themeManager.externalizePopover();
+    // Observe dark / light mode changes
+    themeManager.manageThemeObserver(myChart);
+    // Display the chart using the configured theme and data.
+    myChart.setOption(themeManager.getChartOptions());
+
+  </script>
+</div>
+
+<div class="container-xxl pt-3">
+  <div class="card w-100">
+    <div class="card-body">
+      <h5 class="card-title">Bar specific color</h5>
+      <p class="card-text pe-5">This example shows the example of using two colors for a single series. But this time, we actually keep a single series, displayed once in the legend, while modifying the color of the bars that are below the objective.</p>  
+      <p>To do this, we associate in the list of values the red color with the value below the objectives via the Apache ECharts <ode>itemStyle.color</code> parameter:
+        <code>
+          <pre>
+series: [
+  ...
+  {
+    ...
+    data: [
+      ...
+      {
+        value:results[i],
+        itemStyle: {
+          color: 'var(--ods-functional-5)'
+        }
+      }          
+    ]
+  }
+          </pre>
+        </code>    
+      </p>   
+      <button class="btn btn-icon btn-outline-secondary btn-edit" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Open in playground">
+        <svg width="1.25rem" height="1.25rem" fill="currentColor" aria-hidden="true">
+          <use xlink:href="#lightning-charge-fill" />
+        </svg>
+        <span class="visually-hidden">Open in playground using StackBlitz</span>
+      </button>
+      <div id="two_colors_in_one_in_one_htmlId">
+        <div class="border border-subtle position-relative">
+          <div class="chart_title mx-3">
+            <h4 class="display-4 mx-3 mb-1 mt-3">Title</h4>
+            <h5 class="display-5 mx-3 mb-1 mt-0">Sub-Title</h5>
+          </div>
+          <div id="two_colors_in_one_barLine_holder">
+            <div id="two_colors_in_one_barLine_chart" style="width: 100%; height: 50vh" class="position-relative"></div>
+          </div>
+          <div id="two_colors_in_one_barLine_legend"></div>
+        </div>
+      </div>
+      <script>
+        addViewCode('two_colors_in_one_');
+      </script>
+    </div>
+  </div>
+  <script id="two_colors_in_one_codeId">
+    ///////////////////////////////////////////////////
+    // Used data
+    ///////////////////////////////////////////////////
+
+    var goals = new Array(...new Array(12).keys()).map((i) => {
+      return 50 + Math.random() * 50;
+    });
+
+    var results = new Array(...new Array(12).keys()).map((i) => {
+      return 50 + Math.random() * 50;
+    }).map((res, i) => (res >= goals[i] ? results[i] : {
```

Comment:
This doesn't apply well somehow, but I can't figure out why: 
<img width="1245" height="547" alt="image" src="https://github.com/user-attachments/assets/6f015c05-44d7-44df-aaa2-d85eb35a8a9f" />